community/captains-log/index.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: single
+title:  "Captain's Log"
+sidebar:
+  nav: community
+github_events:
+  image_path: assets/images/graphs/GithubEvents_graph.png
+  alt: "Graph of Github Events Since Open-Sourced"
```

Comment:
```suggestion
  alt: "Graph of GitHub events since open-sourced"
```

Reply:
@dorbin do you think all of the titles should be lower-cased? Is this our style convention?

Reply:
We decided that we're going to use the Google Style guide, and that uses sentence case for titles.